eva-peng/week5/mon/games/app/views/rockpaperscissors/show.html.erb

```diff
@@ -0,0 +1,5 @@
+
+<a href="/">Back to Home</a>
+<hr>
+
+Please enter your Throw (rock, paper, scissors) in the URL following a '/'
```

Comment:
You did an input type for your secret number, so you can do the same with your magic 8 ball question. It is bad practice to have the user type in a question from the browser